database/cassandra/index.js

```diff
@@ -0,0 +1,52 @@
+const cassandra = require('cassandra-driver');
+
+// https://docs.datastax.com/en/developer/nodejs-driver/4.4/features/connection-pooling/
+
+const client = new cassandra.Client({
+  contactPoints: ['127.0.0.1', '127.0.0.1'],
+  localDataCenter: 'datacenter1',
+  keyspace: 'etsy_v02',
+  // pooling: {
+  //   maxRequestsPerConnection: 20000,
+  // },
+});
+
+const t0 = new Date();
+
+
+client.connect()
+  .then(() => {
+    console.log('start', new Date() - t0);
+    const query = 'SELECT sellerid FROM additionalitems WHERE itemid=28999999';
```

Comment:
is this a test query (hardcoded itemid)?

Reply:
Yes, this was to run test queries.